finality-aleph/src/sync/handler.rs

```diff
@@ -206,6 +209,19 @@ impl<I: PeerId, J: Justification, CS: ChainStatus<J>, V: Verifier<J>, F: Finaliz
             false => None,
         };
         self.try_finalize()?;
+        if let Some(id) = &maybe_id {
```

Comment:
This isn't really necessary, as we check the state after taking things out from the task queue anyway.

The comment above should also then be changed to "Returns an ID if this justification was higher than any previously known." or something similar.

Reply:
fixed